docs/docs/typography-js.md

```diff
@@ -64,6 +64,8 @@ const typography = new Typography({
 export default typography
 ```
 
+If you're installing Typography.js into an existing Gatsby Project you've started, you will need to delete all css font styles from your codebase in favor of your new Typography.js settings.
```

Comment:
```suggestion
If you're installing Typography.js into an existing Gatsby project you've started, you will need to delete all conflicting CSS font styles from your codebase in favor of your new Typography.js settings.
```
I'd recommend making it a little more specific! Let me know what you think.

Reply:
@marcysutton  perfect! I like your version.